bot/cogs/sudo.py

```diff
@@ -0,0 +1,119 @@
+import os
+import platform
+import textwrap
+import time
+import traceback
+import typing as t
+from datetime import datetime
+
+from discord import Color, DiscordException, Embed
+from discord import __version__ as discord_version
+from discord.ext.commands import Cog, Context, group
+
+from bot import config
+from bot.core.bot import Bot
+from bot.utils import time as tm
+
+
+class Sudo(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @group(hidden=True)
+    async def sudo(self, ctx: Context) -> None:
+        """Administrative information."""
+        pass
+
+    @sudo.command()
+    async def shutdown(self, ctx: Context) -> None:
+        """Turn the bot off."""
+        await ctx.message.add_reaction("✅")
+        await self.bot.close()
+
+    @sudo.command()
+    async def restart(self, ctx: Context) -> None:
+        """Restart the bot."""
+        await ctx.message.add_reaction("✅")
+        await self.bot.close()
+
+        time.sleep(1)
+
+        os.system("pipenv run start")
```

Comment:
I'm not a big fan of this separation
```suggestion
        await self.bot.close()

        time.sleep(1)
        os.system("pipenv run start")
```